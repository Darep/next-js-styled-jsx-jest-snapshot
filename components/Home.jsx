import React from 'react';
import css from 'styled-jsx/css';

const styles = css`
  #app {
    align-items: center;
    display: flex;
    font-family: sans-serif;
    height: 100vh;
    max-width: 800;
    justify-content: center;
  }

  .welcome {
    margin: 0;
  }
`;

const Home = () => {
  return (
    <div id="app">
      <style jsx>{styles}</style>
      <h1 className="welcome">Welcome to next.js!</h1>
    </div>
  );
};

export default Home;
