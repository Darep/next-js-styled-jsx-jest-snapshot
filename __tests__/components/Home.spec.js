import { shallow } from 'enzyme';
import Home from '../../components/Home';

describe('test', () => {
  it('renders', () => {
    const component = shallow(<Home />);
    expect(component).toMatchSnapshot();
  });
});
