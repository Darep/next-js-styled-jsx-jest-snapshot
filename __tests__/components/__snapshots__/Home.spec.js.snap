// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test renders 1`] = `
<div
  id="app"
>
  <style />
  <h1
    className="welcome"
  >
    Welcome to next.js!
  </h1>
</div>
`;
